### DISCUSSION TITLE
Creative card

### DISCUSSION THREAD

**Discussion by Mahmoud-87** [TDRA-ae/aegov-dls#57] [General]
Creative card is not consider to use the correct font family in the Arabic version, 
currently it show the font: ('Inter', 'Helvetica Neue', ui-sans-serif, system-ui, sans-serif), 
the correct font is: ("Noto Kufi Arabic", ui-sans-serif, "Helvetica Neue", sans-serif)
https://designsystem.gov.ae/docs/components/card
![image](https://github.com/user-attachments/assets/e85d57db-c736-40ca-b6dc-22989bff906a)

**Comment by moiz-conceptlz** [comment]
@Mahmoud-87 I'm not sure how you see the documentation of the component - for this particular page in Arabic? Technical documentation is not translated.

Have you tried to implement this card in your project and check the font applied?

adding @anismansuriui for more context.

**Reply by Mahmoud-87** [reply]
Yes we have implemented the component to MOHRE website
https://mohre.fi.ae/ar/home

**Reply by moiz-conceptlz** [reply]
Converting this discussion to an issue
